src/base/static/components/molecules/place-list-item.js

```diff
@@ -115,13 +118,23 @@ const PlaceFieldText = styled(RegularText)({
   width: "100%",
 });
 
-const PlaceField = ({ field, place, placeFieldConfig }) => {
-  if (field.type === "description") {
+const PlaceField = ({ field, place }) => {
+  const prompt = field.display_prompt || field.prompt || null;
```

Comment:
Are we assuming that if there is no `display_prompt` we will show the field's `prompt` on the list view? I think the `prompt` field is strictly intended to be used on the form, and showing it on the list view (or detail view) might not make sense for some flavors (though for others it probably won't be a big deal).

Reply:
The problem is that we are using `prompt` as if it were a label. For example, the placeholder text on the mywater.world schools dropdown says `School name:`.

Ideally, we would have something like a `label` field. Maybe I'll implement that instead.

Reply:
Sure, we could make what we now call `prompt` into a `label`-- that makes sense.